index.js

```diff
@@ -190,3 +186,5 @@ function validateHelpers(invalidHelpers, relativePath) {
     throw new Error(message);
   }
 }
+
+module.exports = Babel;
```

Comment:
Can you move this up to `module.exports = class BroccoliBabelPlugin extends Filter {` above?